src/hotspot/share/gc/shared/gcArguments.cpp

```diff
@@ -127,6 +127,10 @@ void GCArguments::initialize_heap_flags_and_sizes() {
     }
   }
 
+  if (FLAG_IS_CMDLINE(InitialHeapSize) && InitialHeapSize >= os::physical_memory()) {
```

Comment:
What will happen if the swap space is unlimited?
```
    Memory Soft Limit: Unlimited
    Memory & Swap Limit: Unlimited
```

Reply:
> What will happen if the swap space is unlimited?
> 
> ```
>     Memory Soft Limit: Unlimited
>     Memory & Swap Limit: Unlimited
> ```

Thanks,If the swap space of the physical machine is enabled, it can run
Use Memory & Swap limits to Limit programs